sdk/pinocchio/src/entrypoint/mod.rs

```diff
@@ -258,6 +256,29 @@ macro_rules! process_accounts {
     };
 }
 
+/// Create an `AccountInfo` referencing the same account referenced
+/// by the `AccountInfo` at the specified `index`.
+///
+/// # Safety
+///
+/// The caller must ensure that:
+///   - `accounts` pointer must point to an array of `AccountInfo`s where
+///      the new `AccountInfo` will be written.
+///   - `accounts_slice` pointer must point to a slice of `AccountInfo`s
+///     already initialized.
+///   - `index` is a valid index in the `accounts_slice`.
+#[cold]
```

Comment:
This was needed to stop the compiler trying to optimize the parsing of duplicated accounts and increasing the overall CU consumption.

Reply:
I think it is better to add this explanation as a comment next to the attribute.

Reply:
Good call!